src/Writer/SylkWriter.php

```diff
@@ -146,7 +146,7 @@ private function formatValue($value, $columnIdx, $rowIdx)
 					$value = str_replace(",", ".", $value);
 					break;
 				default:
-					$value = '"' . $value . '"';
+					$value = '"' . (is_array($value) ?(!$value ? print_r($value, true) :"-"): $value) . '"';
```

Comment:
Why do you need to print arrays?

Also a double ternary operator is very unreadable.